src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -187,7 +190,37 @@ export default function DownloadDialog({
     setInstallAllDlcs(!installAllDlcs)
   }
 
-  async function handleInstall(path?: string) {
+  function confirmInstallBrokenAnticheat(path?: string) {
+    showDialogModal({
+      title: t('install.anticheat-warning.title', 'Anticheat Broken/Denied'),
+      message: t(
+        'install.anticheat-warning.message',
+        'The anticheat support is broken or denied. The game will not work. Do you want to install it anyway?'
+      ),
+      buttons: [
+        {
+          text: t(
+            'install.anticheat-warning.install',
+            'Yes (I understand it will not work)'
+          ),
+          onClick: async () => handleInstall(path, true)
```

Comment:
if the user confirms the installation anyway, we call `handleInstall` with the `ignoreAnticheat` arg set to true